package.json

```diff
@@ -393,6 +437,11 @@
                     "when": "explorerResourceIsFolder == true",
                     "group": "zzz_@appServiceDeployCommands"
                 }
+            ],
+            "editor/context": [
+                {
+                    "command": "diagnostics.LogPoints.Toggle"
```

Comment:
should this also have `"when": "config.appservice.logpoints == true`

Reply:
Good catch!